### DISCUSSION TITLE
how to use Bootstrap?

### DISCUSSION THREAD

**Discussion by wraza2k9** [idom-team/idom#742] [General]
How to use bootstrap to style the component?
Can anyone share an example?
Thanks.

**Comment by Archmonger** [comment]
Here's an example of Bootstrap buttons:
https://github.com/idom-team/idom/discussions/526#discussioncomment-1600369

Here's some code I'm using to create bootstrap modals, embedded into a page that already has bootstrap CSS: https://github.com/Archmonger/Conreq/blob/e58d14cdf999d06f7c03daa959916fa2eaf76421/conreq/_core/home/components/modal.py#L26
